src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -350,6 +356,96 @@ private void SetInputPullModePi4(int pinNumber, PinMode mode)
             }
         }
 
+        /// <inheritdoc />
+        protected internal override void SetExtendedPinMode(int pinNumber, ExtendedPinMode altMode)

```

Comment:
argument name should be `extendedPinMode` here and in any similar place like this